ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,137 @@
+const copyStyle = `
+.code-n-btn-wrapper {
+  positon: relative;
+}
+.copy-code-wrapper {
+  display: none;
+  position: absolute;
+  right: 0;
+  margin: 10px 15px 0 0;
+}
+.copy-code-wrapper button {
+   background: #fff;
+   border: none;
+   padding: 10px;
+   cursor: pointer;
+}
+.copy-code-wrapper button i {
+font-size: 1rem;
+}
```

Comment:
Please use tabs, not spaces for indentation.

Reply:
Copy that